### PR TITLE
update readme with details on what the tool does

### PR DESCRIPTION
**Pull request by sandrobonazzola**
## Changes introduced with this PR

The `README.md` file has been updated with details on what the tool does.

---
By contributing to this repository, I agree to the [contribution guidelines](https://github.com/arcalot/.github/blob/main/CONTRIBUTING.md).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,21 @@
-# carpenter
+# Arcaflow Plugin Image Builder
+
+Arcaflow Plugin Image Builder is a tool which has been developed for automatically test, build and publish Arcaflow plugins.
```
Comment (mfleader) [inline README.md:3]: automatically testing, building, and publishing Arcaflow plugins.

**Review by mfleader — CHANGES_REQUESTED** [review]
I like these changes

File: README.md (hunk 1 above)
Comment (sandrobonazzola) [reply README.md]: thanks

File: README.md [hunk 2]
```diff
@@ -1,5 +1,22 @@
 # Carpenter, Arcaflow Plugin Image Builder
 
+Arcaflow Plugin Image Builder is a tool which has been developed for automatically testing, building, and publishing Arcaflow plugins.
+
+More in detail:
+* Python plugins are going to be scanned with pyflakes, unit tested and coverage data for each plugin will be collected.
```
Comment (sanjacodes) [inline README.md:6]: Hard to read, suggest either:

```
Python plugins are going to be scanned with pyflakes. Unit tests and coverage data for each plugin will be collected.
```

OR

```
Python plugins are going to be unit tested, scanned with pyflakes, and coverage data for each plugin will be collected.
```

File: README.md [hunk 3]
```diff
@@ -1,5 +1,22 @@
...
+  Successfully tested plugins will be published on pypi registry automatically on tag event.
+* Go plugins are going to be unit tested and coverage data for each plugin will be collected.
+
+Successfully tested plugins will be also added to docker images and end to end tested where possible.
+Successful tested images will be published to quay.io automatically on tag event.
```
Comment (sanjacodes) [inline README.md:11]: Successfully

**Review by sanjacodes — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (sandrobonazzola) [reply README.md:6]: thanks

File: README.md (hunk 3 above)
Comment (sandrobonazzola) [reply README.md:11]: thanks

**Review by sanjacodes — APPROVED** [review]

**Pull request merged**